testers/testers/pyta/specs/settings_schema.json

```diff
@@ -66,12 +75,19 @@
             "uniqueItems": true
           },
           "timeout": {
+            "title": "Timeout",
             "type": "integer",
             "default": 30
           },
           "feedback_file_name": {
+            "title": "Feedback file",
             "type": "string"
           },
+          "upload_annotations": {
```

Comment:
Right now, uploading annotations requires that we specify a feedback file as well. This is why it was in a dependency. I agree that we should be able to upload annotations without requiring a feedback file as well but that will require some changes to the tester code. 

I would suggest keeping upload_annotations as a dependency for feedback_file_name for now and then we can change this in a different PR.